src/app/space/create/deployments/resource-usage/utilization-bar.component.ts

```diff
@@ -21,15 +21,31 @@ export class UtilizationBarComponent implements OnInit {
   total: number;
   usedPercent: number;
   unusedPercent: number;
+  color: string;
+
+  private colors = {
+    'Okay': '#39a5dc', // Light blue
```

Comment:
Why aren't we using styles for this. @mindreeper2420 can you take a look.

Reply:
@jiekang can you have the file call a class, and then define the colors in those classes? The hex symbols are also incorrect, as they do no follow the PatternFly palette.
http://www.patternfly.org/styles/color-palette/

Reply:
@mindreeper2420 To clarify: class as in html class defined in a external css file? It should be possible to dynamically apply a class to an html element with angular; ngClass rings a bell.

As far as the colors, the Okay one was from the patternfly default and I think it exists as pf-blue-300. For the orange, I used gimp's color picker on the mock image to find an exact but probably inaccurate hex as I didn't see any prior examples to use. I will use pf-orange-400: #ec7a08 instead. Sorry about that, I wasn't aware of the palette. Thanks!

As an aside, I feel like there should be a way to access the patternfly palette and use that, but I'll research that in a separate issue/PR.

Reply:
Oh; it looks like the warning hex does also exist as pf-orange-300 '#f39d3c' on http://www.patternfly.org/styles/color-palette/

Reply:
@jiekang If you can use ngClass to reference an external CSS class on the element, that'd be great.

As far as accessing the color palette, all of the colors use variables to select them, i.e. `@color-pf-orange-400` will call the hex code for that color orange. Right now, there are no extensions for VSCode that allow for autocomplete of these, but there used to be one for Atom (not sure if it is still maintained). Nevertheless, all of the colors in the palette can be referenced with the `@color-pf-` variable.